exporter/instanaexporter/config.go

```diff
@@ -31,6 +31,8 @@ type Config struct {
 
 	AgentKey string `mapstructure:"agent_key"`
 
+	CAFile string `mapstructure:"ca_file" json:"omitempty"`
```

Comment:
Isn't this part of the HTTP Client Settings already? More specifically, of the TLS Settings:

https://github.com/open-telemetry/opentelemetry-collector/blob/bb193a24d0a968027ad48c397899ae3688165756/config/configtls/configtls.go#L39-L42

It would then be used as:

```yaml
instana:
  endpoint:
  tls:
    ca_file: "path-to-ca-file"
```

See a more complete example, where "otlp" can be replaced by "instana": https://github.com/open-telemetry/opentelemetry-collector/tree/bb193a24d0a968027ad48c397899ae3688165756/config/configtls#client-configuration

Reply:
Thanks for the review, @jpkrohling .
I've applied a couple of changes. Could you please take a look one more time?